phpstan.neon

```diff
@@ -0,0 +1,19 @@
+parameters:
+  customRulesetUsed: true
+  reportUnmatchedIgnoredErrors: false
+  # Ignore phpstan-drupal extension's rules.
+  ignoreErrors:
+    - '#\Drupal calls should be avoided in classes, use dependency injection instead#'
+    - '#Plugin definitions cannot be altered.#'
+    - '#Missing cache backend declaration for performance.#'
+    - '#Plugin manager has cache backend specified but does not declare cache tags.#'
+    #TODO: Remove when https://www.drupal.org/project/drupal/issues/2367933 is fixed.
```

Comment:
This is fixed - but I guess you mean in all supported D8 branches. Atm it is only fixed in 8.8.x.